Dockerfile.dummy_presenter

```diff
@@ -0,0 +1,12 @@
+FROM python:3.8.1
```

Comment:
Cool, can you move this Dockerfile into the dummy_presenter folder?

Reply:
Done.